libs/payments/ui/src/lib/client/components/MetricsWrapper.tsx

```diff
@@ -0,0 +1,24 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use client';
+
+import * as Sentry from '@sentry/nextjs';
+import { WithContextCart } from '@fxa/payments/cart/cart.types';
+import { useEffect } from 'react';
+
+export function MetricsWrapper({
+  cart,
+  children,
+}: {
+  cart?: WithContextCart;
+  children: React.ReactNode;
+}) {
+  useEffect(() => {
+    cart &&
+      Sentry.getGlobalScope().setTag('metricsOptedOut', cart.metricsOptedOut);
+  }, [cart]);
```

Comment:
[This doesn't need to hold up merging this PR] Could this be narrowed down to just `metricsOptedOut` instead of changes on the entire cart?

Reply:
`cart?.metricsOptedOut` is strictly better here :) updating